man/loader.conf.xml

```diff
@@ -399,6 +399,17 @@ sbvarsign --attr "${attr}" --key KEK.key --cert KEK.pem --output db.auth db db.e
 
```

Comment:
hmm, i am trying to understand the UI choices you made. the [1/1] thing, what is that supposed to convey? the 2nd number is the number of entries, right? but the first one?

I think using left/right to navigate in the group is a bit unexpected no? i don#t know of any other menu that works like that? did you have prior art in mind?

if figure the most obvious way would be that you first hit enter on the submenu and then ob the entry on it?

the way i always though such grouping should look like in the end is like this to show the newest entry of each group directly in the entry, and then immediately after show a "More..." entry that gives you access to the older stuff.

hence in your example above you'd have:

```
Arch Linux
 ↳ more...
Fedora
 ↳ more...
openSUSE
 ↳ more...
Reboot Into Firmware Interface
```

and maybe even grey out the "more..." lines.

Reply:
> hmm, i am trying to understand the UI choices you made. the [1/1] thing, what is that supposed to convey? the 2nd number is the number of entries, right? but the first one?

The 1st number is the entry selected within the group. In the example shown in the description, `[1/4] > openSUSE Tumbleweed (6.6.1-3.gfd0e37e-default)`, clicking enter will boot the Tumbleweed entry shown.

> I think using left/right to navigate in the group is a bit unexpected no? i don#t know of any other menu that works like that? did you have prior art in mind?

No, but I found that using just the arrows to move between groups (up/down) and entries in the same group (right/left) is very intuitive (at least for me).